src/components/RoomNameInput.js

```diff
@@ -73,7 +73,7 @@ class RoomNameInput extends Component {
         const target = nativeEvent.target;
         const selection = target.selectionStart;
         const modifiedRoomName = this.modifyRoomName(roomName);
-        this.setState({roomName: modifiedRoomName}, () => {
+        this.setState({roomName: `${CONST.POLICY.ROOM_PREFIX}${roomName}`}, () => {
```

Comment:
@eVoloshchak can't we remove `value` prop from `TextInput`, and the entire selection logic?

Reply:
> @eVoloshchak can't we get rid of the `value` prop from `TextInput`, roomName from state, and the entire selection logic?

We can, I've updated the PR